admin-ui/src/components/Widget/ProductReviewWidget/ProductWidgetColorSetting.tsx

```diff
@@ -131,473 +119,7 @@ const ProductWidgetColorSetting = ({ name }: { name: string }) => {
-                          floodOpacity="0"
-                          result="BackgroundImageFix"
-                        ></feFlood>
-                        <feBlend
-                          mode="normal"
-                          in="SourceGraphic"
-                          in2="BackgroundImageFix"
-                          result="shape"
-                        ></feBlend>
-                        <feGaussianBlur
-                          stdDeviation="5.48184"
-                          result="effect1_foregroundBlur_5848_86970"
-                        ></feGaussianBlur>
-                      </filter>
-                      <filter
-                        id="filter11_f_5848_86970"
-                        x="-1.70392"
-                        y="8.86334"
-                        width="27.7465"
-                        height="37.4296"
-                        filterUnits="userSpaceOnUse"
-                        colorInterpolationFilters="sRGB"
-                      >
-                        <feFlood
-                          floodOpacity="0"
-                          result="BackgroundImageFix"
-                        ></feFlood>
-                        <feBlend
-                          mode="normal"
-                          in="SourceGraphic"
-                          in2="BackgroundImageFix"
-                          result="shape"
-                        ></feBlend>
-                        <feGaussianBlur
-                          stdDeviation="4.47507"
-                          result="effect1_foregroundBlur_5848_86970"
-                        ></feGaussianBlur>
-                      </filter>
-                      <filter
-                        id="filter12_f_5848_86970"
-                        x="-25.0136"
-                        y="-3.50118"
-                        width="44.6238"
-                        height="51.7582"
-                        filterUnits="userSpaceOnUse"
-                        colorInterpolationFilters="sRGB"
-                      >
-                        <feFlood
-                          floodOpacity="0"
-                          result="BackgroundImageFix"
-                        ></feFlood>
-                        <feBlend
-                          mode="normal"
-                          in="SourceGraphic"
-                          in2="BackgroundImageFix"
-                          result="shape"
-                        ></feBlend>
-                        <feGaussianBlur
-                          stdDeviation="8.22276"
-                          result="effect1_foregroundBlur_5848_86970"
-                        ></feGaussianBlur>
-                      </filter>
-                      <clipPath id="clip0_5848_86970">
-                        <rect width="24" height="24" rx="4" fill="white"></rect>
-                      </clipPath>
-                    </defs>
-                  </svg>
+                  <i className="farp farp-custom !frt-text-3xl"></i>
```

Comment:
<img width="198" alt="Screenshot 2024-11-19 at 5 28 54 AM" src="https://github.com/user-attachments/assets/c1be5cb9-8bb6-4b4e-8f14-8cf7672f02d7">

seems both are same colos, could you please differentiate.